cmd/secrets-provider/main.go

```diff
@@ -51,51 +52,55 @@ func main() {
 		printErrorAndExit(messages.CSPFK009E)
 	}
 
-	// Configure exponential backoff
-	expBackoff := backoff.NewExponentialBackOff()
-	expBackoff.InitialInterval = 2 * time.Second
-	expBackoff.RandomizationFactor = 0.5
-	expBackoff.Multiplier = 2
-	expBackoff.MaxInterval = 15 * time.Second
-	expBackoff.MaxElapsedTime = 2 * time.Minute
+	syncIntervalTime := 3
+	if envSyncIntervalVal := os.Getenv("SYNC_INTERVAL_TIME"); envSyncIntervalVal != "3" && envSyncIntervalVal != "" {
+		syncIntervalTime, _ = strconv.Atoi(envSyncIntervalVal)
+	}
```

Comment:
it is best to move this to the config file. that is the place where we read the env to set the config

Reply:
Why do you need comparison to 3 here ? Atoi will handle it anyway, right ?